Frontend/ui-driver/src/Screens/UploadAdhaarScreen/Controller.purs

```diff
@@ -67,6 +68,13 @@ data Action = BackPressed
             | UploadImage
             | PreviewImageAction
 
+uploadFileConfig :: UploadFileConfig
```

Comment:
Try not to have redundant code, As this config is being used in multiple files. We can keep it in a common file from where it can be imported to all required places